### DISCUSSION TITLE
How to track metrics from the first epoch?

### DISCUSSION THREAD

**Discussion by khanhtq-eureka** [mlflow/mlflow#6763] [Q&A]
I use pytorch lightning with config ```log_every_n_steps=1``` but metrics in mlflow only show up when first epoch completes. Is there anyway to get the metrics right after the first step of the first epoch?

**Comment by JHibbard** [comment]
Hi khanhtq-eureka,

Could you share your code and elaborate on how you'd like to access metrics? I'll take a look :)

Thanks!
Jim
